addons/website/static/src/components/dialog/page_properties.js

```diff
@@ -151,6 +154,19 @@ export class DuplicatePageDialog extends Component {
     }
 }
 
+export class PagePropertiesFormController extends FormController {
+    static props = {
+        ...FormController.props,
+        clonePage: { type: Function, optional: true },
+        deletePage: { type: Function, optional: true },
```

Comment:
Is it optional though? If you don't pass those, won't you have a button to clone/delete the page but which does nothing?

Reply:
@qsm-odoo Indeed, `clonePage: Function, deletePage: Function` will do the job.

Reply:
@qsm-odoo The commit is amended